projects/application/dweb-search-en.md

```diff
@@ -0,0 +1,53 @@
+
+## Project Name Dweb Search
+![Dweb Search](https://raw.githubusercontent.com/anwen/dweb-search/main/docs/dweb-search-1.png)
+
+## Category 
+application
+
+## Project Description
+Dweb Search is a decentralized Search Engine for IPFS and Filecoin.
+
+The platform also provides free snapshot and API, reward creators and curators who submit metadata.
+
+## Project Status
+alpha
+
+## Previews
+![Homepage](https://raw.githubusercontent.com/anwen/dweb-search/main/docs/assets/preview-1.png)
+
+## Target Audience
+Filecoin miners, Storage Clients, everyone who want to search the dweb
+
+## Rough estimated user base (if applicable)
+about 10 users
+
+## Github repo
+https://github.com/anwen/dweb-search
+
+## Website
+https://dweb-search.on.fleek.co/
+
+## Docs
+https://github.com/anwen/dweb-search/wiki
+
+## Team Info
+
+### Team Size  
+1+
+
+### Team members  
+- Askender: a dweb developer
+- Everyone who join [Discord: Dweb-Dev](https://discord.gg/QaEwmJMDJ2) can be a team member
+
+## How the community can engage
+GitHub Discussion: https://github.com/filecoin-project/community/discussions/180
```

Comment:
Could you also please reformat this part as well instead of putting them all in one paragram?